packages/eslint-config/test/__snapshots__/eslint.spec.js.snap

```diff
@@ -484,7 +484,8 @@ Object {
       "off",
     ],
     "curly": Array [
-      0,
+      "error",
+      "all",
```

Comment:
Not related to this PR, but I wonder if we should split the snapshots in several files so it is easier to see what's going on... 🤔 

cc: @deini